src/components/Input/Input.scss

```diff
@@ -276,4 +284,4 @@
     margin: 0 3px;
     opacity: 0.6;
   }
-}
+}
```

Comment:
don't think the extra space at the bottom of the page is needed?

Reply:
This is still at the end of the file, with no extra space. It is just that it has moved down the file from line 279 to 287

Reply:
I think that fixed it